internal/services/postgres/postgresql_server_resource.go

```diff
@@ -214,13 +214,14 @@ func resourcePostgreSQLServer() *pluginsdk.Resource {
 			"ssl_minimal_tls_version_enforced": {
 				Type:         pluginsdk.TypeString,
 				Optional:     true,
-				Default:      string(servers.MinimalTlsVersionEnumTLSOneTwo),
+				Default:      string(servers.MinimalTlsVersionEnumTLSEnforcementDisabled),
```

Comment:
Hello Tom, I was thinking that as well, I should have referenced mysql since it's similar. The changes have been reverted and only the doc is being updated.